### PR TITLE
Fix #363: Can't open files that specify the OASIS schema.

### LINKED ISSUE #363: Cannot load sarif files with schema from docs.oasis-open.org
https://docs.oasis-open.org/sarif/sarif/v2.1.0/cos02/schemas/sarif-schema-2.1.0.json

snippet 

```json
{
	"version": "2.1.0",
	"$schema": "https://docs.oasis-open.org/sarif/sarif/v2.1.0/cos02/schemas/sarif-schema-2.1.0.json",
	"runs": [...]
}
```

changing this to https://raw.githubusercontent.com/oasis-tcs/sarif-spec/master/Schemata/sarif-schema-2.1.0.json works 

Thanks for your time

### PR DESCRIPTION
**Pull request by lgolding**

### CODE REVIEW

File: src/extension/update.spec.ts [hunk 1]
```diff
@@ -83,7 +83,7 @@ const makeStubs = () => ({
             executeCommand: fake() // (command: string, ...rest: any[]) => undefined
         },
         extensions: {
-            getExtension: () => ({ packageJSON: { version: '3.0.0' } })
+            getExtension: () => ({ packageJSON: { version: '3.0.1' } })
```
Comment (lgolding) [inline src/extension/update.spec.ts:86]: >1 [](start = 64, length = 1)

I don't know the right way to increment the version.

Comment (jeffersonking) [reply src/extension/update.spec.ts:86]: This is a unit test for the auto-update feature. It's mock data and does not need to be updated.

The version source of truth is in `package.json`.

**Review by jeffersonking — APPROVED** [review]
Just the one comment.

File: src/extension/update.spec.ts (hunk 1 above)
Comment (lgolding) [reply src/extension/update.spec.ts:86]: Thanks!

---
In reply to: [509609462](https://github.com/microsoft/sarif-vscode-extension/pull/364#discussion_r509609462) [](ancestors = 509609462)

**Pull request merged**